packages/flutter_test/lib/src/binding.dart

```diff
@@ -1489,6 +1496,10 @@ class LiveTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
     assert(inTest);
     assert(!_expectingFrame);
     assert(_pendingFrame == null);
+    if (framePolicy == LiveTestWidgetsFlutterBindingFramePolicy.benchmarkLive) {
+      // Ignore all pumps and just wait.
+      return Future<void>.delayed(duration?? Duration.zero);
```

Comment:
nit: space between `duration ??`.

Why are we awaiting a delay of the duration before stting a timer for the duration again below? Won't that mean we get double the duration expected?

Reply:
It's an early return. When the policy is `benchmarkLive`, the below part is not executed.

Reply:
Ahh sorry, I missed the scope of the return here.